### ISSUE TITLE
Unable to open simba gui

### ISSUE THREAD

**Issue by jacobk14** [sgoldenlab/simba#41]
I have install Simba using pip install simba-uw-no-tf, but when I tried launching Simba, I got the traceback below. I know this it is successfully installed because I opened the directory in my terminal (also shown below). I am using a Linux virtual machine is that changes anything. 
![image](https://user-images.githubusercontent.com/66270143/86048001-caf39380-ba1d-11ea-826e-4b219ee102a9.png)
![image](https://user-images.githubusercontent.com/66270143/86048138-fe362280-ba1d-11ea-9611-2cd95debeac1.png)

**Comment by sgoldenlab** [comment]
Hi @jacobk14 - first thanks for trying SimBA! However, for the moment, we only support Windows use. This is a known issue with the encoded windows path preventing SimBA to boot on Linux / MacOS. Even if you do get past this specific issue, there are many path readings in SimBA that are designed for Windows making it difficult to work with in Linux at the moment. 

Please see these closed issues for some related info:

https://github.com/sgoldenlab/simba/issues/37

https://github.com/sgoldenlab/simba/issues/39

For now, I recommend getting hold of a Windows machine and run SimBA on it. You mention you are running Linux on a virtual machine. If this happens to be on native Windows, I'd suggest exiting the virtual machine and installing/running SimBA in native Windows.

**Comment by sgoldenlab** [comment]
Hi @jacobk14 - I've uploaded a Linux compatible version here - it's very much work in development but most of the function (excluding some of the video pre-processing functions) should work in Linux. If you do decide to try it out and encounter some bugs it would be great if you report them so I can fix them asap

`pip install simba-uw-tf-dev`

**Comment by akh8zm** [comment]
Hi! First, thanks so much for the work in progress Linux version! I'm trying to use it now and running into a few kinks regarding install on an Ubuntu virtual machine (through MATE) when I try to run `simba`.  The first two were straightforward enough: didn't find module named `numba` so I ran `pip install numba`; then couldn't find the file `TheGoldenLab.png`, so I manually found the file and changed the name from `TheGoldenLab.PNG` to lowercase. 
Now I'm getting another error after the GoldenLab image pops up:
`_tkinter.TclError: couldn't recognize data in image file "/sfs/qumulo/qhome/akh8zm/.conda/envs/simba-linux/lib/python3.6/site-packages/simba/golden.png" `
I've verified that the file does exist and the properties look ok to me:
<img src="https://user-images.githubusercontent.com/56124630/88329709-649a2200-ccf8-11ea-9739-5d552751ae94.png" width=30% height=30%>
<img src="https://user-images.githubusercontent.com/56124630/88329746-6cf25d00-ccf8-11ea-9748-48f560315f04.png" width=30% height=30%>
Not sure about next steps. Some issues I've found online say that tkinter can only support GIFs but that would require changing the filename.
